images/dns-controller-builder/Dockerfile

```diff
@@ -12,16 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:jessie
+FROM gcr.io/google_containers/debian-base-amd64:0.1
```

Comment:
yes, I think bash was removed in latest version of debian-base-amd64. Let me test with the new version